services/hanke-service/src/main/resources/application.properties

```diff
@@ -19,59 +18,44 @@ haitaton.gdpr.audience=${HAITATON_GDPR_AUDIENCE:http://localhost:8080/haitaton}
 haitaton.gdpr.authorization-field=${HAITATON_GDPR_AUTHORIZATION_FIELD:http://localhost:8080}
 haitaton.gdpr.query-scope=${HAITATON_GDPR_QUERY_SCOPE:haitaton.gdprquery}
 haitaton.gdpr.delete-scope=${HAITATON_GDPR_DELETE_SCOPE:haitaton.gdprdelete}
-#logging.level.org.springframework.security=DEBUG
-
 # Disable endpoints that are e.g. in development and should not be in production.
 haitaton.features.hanke-editing=${HAITATON_FEATURE_HANKE_EDITING:false}
 haitaton.features.user-management=${HAITATON_FEATURE_USER_MANAGEMENT:false}
-
 # For dev and test environments, enable the testdata controller for resetting data
 haitaton.testdata.enabled=${HAITATON_TESTDATA_ENABLED:false}
-
 spring.datasource.url=jdbc:postgresql://${HAITATON_HOST:localhost}:${HAITATON_PORT:5432}/${HAITATON_DATABASE:haitaton}
 spring.datasource.username=${HAITATON_USER:haitaton_user}
 spring.datasource.password=${HAITATON_PASSWORD:haitaton}
-
 # JPA
 spring.jpa.open-in-view=false
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 # This makes the database field names to match the entity member names
 spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 # Don't log SQL queries. This is the default, but left here to make it easy to enable while developing.
 spring.jpa.show-sql=false
-
 spring.servlet.multipart.max-file-size=100MB
 spring.servlet.multipart.max-request-size=101MB
-
 # Spring Boot Actuator Management for Kubernetes (see https://docs.spring.io/spring-boot/docs/2.3.4.RELEASE/reference/html/production-ready-features.html#production-ready-kubernetes-probes)
 # Use separate HTTP port for probes
 management.server.port=8081
 management.endpoint.health.probes.enabled=true
 management.health.livenessState.enabled=true
 management.health.readinessState.enabled=true
 # Spring Boot Actuator Info properties
-info.java-version = ${java.specification.version}
-info.java-vendor = ${java.specification.vendor}
+info.java-version=${java.specification.version}
+info.java-vendor=${java.specification.vendor}
 server.forward-headers-strategy=FRAMEWORK
-
```

Comment:
I don't think removing empty lines from between sections is helping the readability.

Reply:
Added line breaks back.

Reply:
All the useful edits were removed as well.